### DISCUSSION TITLE
package-local nicknames

### DISCUSSION THREAD

**Discussion by fstamour** [fstamour/cl-car#11] [Ideas]
From https://gist.github.com/fstamour/bacda529ac73a05505e98b43ea2d4fc8?permalink_comment_id=4774127#gistcomment-4774127 :

> There should be a CDR for package-local nicknames. (see https://github.com/Gleefre/cdr-package-local-nicknames/ for a draft I wrote some time ago -- a rewrite is probably needed)

**Comment by Gleefre** [comment]
I have revisited my old draft, here's the link again: https://gleefre.github.io/cdr-package-local-nicknames/index.html. Any feedback is welcome :)
